apps/glusterfs/app_volume.go

```diff
@@ -41,6 +42,16 @@ func (a *App) VolumeCreate(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Check group id
+	switch {
+	case msg.Gid < 0:
+		http.Error(w, "Bad group id less than zero", http.StatusBadRequest)
+		return
+	case msg.Gid >= math.MaxInt32:
```

Comment:
I thought that also, but then I noticed that Kubernetes has a test against math.MaxInt32:

https://github.com/kubernetes/kubernetes/blob/master/pkg/api/validation/validation_test.go#L3238

So I decided to follow their lead.